prime-dotnet-webapi/Configuration/AccessTerms/AgreementConfiguration.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using System.Linq;
+using System.Collections.Generic;
+
+using Prime.Models;
+using Prime.Configuration.Resources;
+
+namespace Prime.Configuration.Agreements
+{
+    public static class AgreementConfiguration
```

Comment:
Despite being named like a configuration file, its not actually an IEntityTypeConfiguration. It just holds the seed data for all the agreements in one place.